src/python/ensembl/utils/rloader.py

```diff
@@ -0,0 +1,73 @@
+# See the NOTICE file distributed with this work for additional information
+#   regarding copyright ownership.
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#       http://www.apache.org/licenses/LICENSE-2.0
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""
+Allow to seamlessly load / read the content of a remote file as if it was located locally.
+
+@author: Marc Chakiachvili
+
+"""
+import configparser
+import json
+import logging
+from io import StringIO
+
+import dotenv
+import requests
+import requests.exceptions as exc
+import yaml
+
+# Add exception processing
+# File does not exists.
+# Return as dict ?
+# Add Loading parser ?
+logger = logging.getLogger(__name__)
+
+
+class RemoteFileLoader(object):
+    format = ('yaml', 'ini', 'env', 'json')
+
+    def __init__(self, parser=None) -> None:
+        super().__init__()
+        self.format = parser
+
+    def __parse(self, content):
+        if self.format == 'yaml':
+            return yaml.load(content, yaml.SafeLoader)
+        elif self.format == 'ini':
+            config = configparser.ConfigParser()
+            try:
+                config.read_string(content)
+            except configparser.MissingSectionHeaderError:
+                content = StringIO("[top]\n" + content).read()
```

Comment:
File-like object `StringIO` is not needed here as `read_string()` already reads a string. Also to be consistent with `ConfigParser()` the added section should probably be "DEFAULT".
```suggestion
                content = "[DEFAULT]\n" + content
```